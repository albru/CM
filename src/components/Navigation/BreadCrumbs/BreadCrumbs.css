.BreadCrumbs {
    position: absolute;
    top: 0;
    left: 0;
    width: 95%;
    font-size: 12px;
    display: flex;
    flex-flow: row;
    justify-content: flex-start;
}