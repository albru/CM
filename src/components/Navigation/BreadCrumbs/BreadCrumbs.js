import React from 'react';
import BreadCrumb from './BreadCrumb/BreadCrumb';
import classes from './BreadCrumbs.css'

const breadCrumbs = props => {
    const crumbs = props.crumbs.map((item, index) => {
        return (
            <BreadCrumb       
                name={item.crumbName}
                href={item.crumbHref}
                type={item.linkType}
                key={index}
            />
        )
    })
    return (
        <div className={classes.BreadCrumbs}>
            {crumbs}
        </div>
    )
};


export default breadCrumbs;