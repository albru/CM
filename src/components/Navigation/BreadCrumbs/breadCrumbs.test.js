import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import BreadCrumbs from './BreadCrumbs';

configure({ adapter: new Adapter() })

describe('BreadCrumbs', () => {
    it('should render correctly', () => {
        const props = {
            crumbs: []
        }
        const output = shallow(
            <BreadCrumbs {...props}/>
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});