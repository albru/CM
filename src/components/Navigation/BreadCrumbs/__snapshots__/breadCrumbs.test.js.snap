// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadCrumbs should render correctly 1`] = `<div />`;
